runEval.Rmd

```diff
@@ -15,160 +15,166 @@ set.seed(4857)
 
-pull_section <- function(section, file) {
-  print = FALSE
-  return_string = ''
-  for (line in readLines(file)) {
-    if (line == section) {
-      print = !print
-    }
-    if (print) {
-      return_string = paste0(return_string, line, '\n')
-    }
-  }
-  return(return_string)
-}
+```{r}
+cat(pull_section('# dependencies', here::here('R','runEval.R')))
+```
 
-cat(pull_section('# section 1', here::here('R','runEval.R')))
+## Setting up the `run_df` data frame that controls the experiments
+```{r}
+cat(pull_section('# setup run_df', here::here('R','runEval.R'), num_lines = TRUE))
+```
+The `run_df` is a dataframe where each row will be a complete experiment configuration. The above functions add information relevant to their role. The source is very straightforward should one want to modify for their own purposes. In more detail:
 
+```{r}
+cat(pull_section('setup_run_df', here::here('R','runEval.R'), num_lines = TRUE))
 ```
-Above the `template_df` is initialized with column names and global parmeters, `n_days` and  `n_pop`. The simulation will populate the data columns, actual data may only populate some of the columns, e.g., `template_df$d` and `templte_df$tweets` in the case of the current Brazil data. Printing the data frame: 
+Running code up to this line shows the following values, note that the dataframe is transposed with rows as columns and columns as rows--unreadable otherwise.
+
+```{r echo=FALSE}
+# dependencies
+library(tidyverse)
+library(cmdstanr)
+library(data.table)
+library(kableExtra)
+
+source(here::here("R","util.R"))
+source(here::here("R","SIRTDsim.R"))
+source(here::here("R","sim_configs.R"))
+source(here::here("R","modeling_configs.R"))
+# dependencies
+# setup run_df
+run_df <- setup_run_df(seed = 93435, n_pop = 1000, n_days = 7)
+kable(t(run_df))
+```
+
+This line constructs a basic template for the `run_df`, there is just one row at this point--remember transposed output. All subsequent additions/elaborations draw from this dataframe. 
 ```{r}
-print(template_df)
+cat(pull_section('sim_brazil_1', here::here('R','runEval.R'), num_lines = TRUE))
 ```
+The `sim_brazil_1` is a parameterization that roughly mimics the shape of the COVID-19 outbreak with corresponding simulated values as determined by trying values by hand. Look at the source for more info. Note that the prefix on the function is 'sim', which means that it is part of simulation. There is still one row in the dataframe but with simulation data and the generating paramaterzation. 
 
-### Adding simulation parameters
+The, remember transposed, dataframe is:
```

Comment:
Aren't you being pedantic? This is the third time you mention "transposed" in less than 10 lines?